.gitignore

```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+lib/src/.DS_Store
```

Comment:
Huh, not sure why we have all these in the .gitignore.. i probably chose the wrong template.  can you just replace the whole thing with this and we'll see how it goes..
```
node_modules/
.DS_Store
logs
*.log
```

Reply:
Cleaned .gitignore